content/docs/index.md

```diff
@@ -4,7 +4,33 @@ Welcome! In here you may find all the guiding material and technical documents
 needed to learn about DVC: how to use it, how it works, and where to go for
 additional resources.
 
-## Before you start
+<cards>
+
+<card href="/doc/tutorials/get-started" heading="Get Started">
+
+A step-by-step introduction into basic DVC features
+
+</card>
+
+<card href="/doc/user-guide" heading="User Guide">
+
+Study the detailed inner-workings of DVC in its user guide
```

Comment:
Super minimal: Missing period. I'll add it no worries. Same in cmd ref card text